pyfr/solvers/acnavstokes/kernels/flux.mako

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+<%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
+
+% if ndims == 2:
```

Comment:
It should be easy to unify the `ndims = 2` and `ndims = 3` cases for this kernel (like we do for the inviscid flux).